src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -154,14 +155,23 @@
                         this.defaultMessage = "An error occurred fetching Git commits";
                     });
             },
-            getParameterValues(values) {
+            getParameterValues(values, isValid) {
                 if (values) {
                     this.parameterValues.forEach((param, key) => {
                         if (values[key].name == param.name) {
                             param.value = values[key].value
                         }
                     })
                 }
+                this.validateParams(isValid)
+            },
+            validateParams(isValid) {
+                this.disableRun = false
```

Comment:
Feels like `disableRun` should actually be a computed property e.g. `!this.paramError || this.runningStatus`. If it was, and any `paramError` was populated from `<parameter-list>` then the function might not even be necessary.

Reply:
Yes that would work since we clear the status when anything changes